apps/hash-graph/lib/graph/src/snapshot.rs

```diff
@@ -257,9 +260,9 @@ impl<C: AsClient> SnapshotStore<C> {
                         SqlState::FOREIGN_KEY_VIOLATION => {
                             report.attach_printable(StatusCode::NotFound)
                         }
-                        SqlState::UNIQUE_VIOLATION => {
-                            report.attach_printable(StatusCode::AlreadyExists)
-                        }
+                        SqlState::UNIQUE_VIOLATION => report
+                            .attach_printable(StatusCode::AlreadyExists)
+                            .attach(VersionedUrlAlreadyExists),
```

Comment:
Is this only for types? Can snapshots not return entities too?

Reply:
Good point, I forgot that it's may be possible to insert an entity with the same UUID in snapshots.
[`09e51d5` (#2638)](https://github.com/hashintel/hash/pull/2638/commits/09e51d5e59d13e19ca9c8155470a6f0327101295)